src/main/java/com/firebolt/FireboltDriver.java

```diff
@@ -1,27 +1,28 @@
 package com.firebolt;
 
 import com.firebolt.jdbc.connection.FireboltConnection;
-import com.firebolt.jdbc.exception.FireboltSQLFeatureNotSupportedException;
+import com.firebolt.jdbc.util.LoggerUtil;
 import com.firebolt.jdbc.util.PropertyUtil;
 import com.firebolt.jdbc.util.VersionUtil;
-import lombok.CustomLog;
 
 import java.sql.Connection;
 import java.sql.Driver;
 import java.sql.DriverPropertyInfo;
 import java.sql.SQLException;
-import java.sql.SQLFeatureNotSupportedException;
 import java.util.Properties;
 import java.util.logging.Logger;
 
-@CustomLog
 public class FireboltDriver implements Driver {
 
 	public static final String JDBC_FIREBOLT = "jdbc:firebolt:";
+	private static Logger root;
```

Comment:
Let's have this as `root_log` instead. Otherwise root is too generic to be meaningful when found in the code.